web_external/stylesheets/body/featuresetPage.styl

```diff
@@ -1,4 +1,40 @@
-.isic-featureset-global-features-content
-.isic-featureset-local-features-content
-  display inline-block
-  width 800px
+.isic-featureset-container
+
+  .isic-featureset-table
+    width 100%
+
+    .isic-featureset-table-label
+      font-weight bold
+      width 100px
+
+    .isic-featureset-table-section-header
+      background-color #eee
+      padding 0px 8px
+      font-size 20px
+
+    .isic-featureset-table-feature-header
+      border 0px
+      padding-top 10px
+      font-size 20px
```

Comment:
This still appears slightly emphasized at `16px`, and takes up a lot less space.